commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -23,11 +25,13 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
       commercestorepricing.PricingRequestItem item = items.get(i);
       products.add(item.getProductId());
     }
-
+    Map<String, PriceDataFromExternalService> pricesFromExternalService;
     // Get prices from an external service
-    Map<String, PriceDataFromExternalService> pricesFromExternalService = getPricesFromExternalService(
-      products
-    );
+    if(useHTTPService){
+      pricesFromExternalService = getPricesFromExternalService(products);
+    }else{
+      pricesFromExternalService = getPriceFromStaticResponse(products);
```

Comment:
Here we are calling getPriceFromStaticResponse with productIds, yet the getPriceFromStaticResponse accepts skus. We should update getPriceFromStaticResponse to take in productIds rather than skus.

Reply:
This was because of the comment in this file which mentioned
// If the request is successful, parse the JSON response; The response includes the negotiated
    // as well as list price for each SKU. It looks something like this: response =
    // {"ProductID1":{"listPrice":10,"unitPrice":5}};
    
    If this is meant to be IDs we can modify this @jwaref